nyaa/templates/layout.html

```diff
@@ -90,6 +90,7 @@
 						{% endif %}
 						{% if g.user.is_moderator %}
 							<li><a href="{{ url_for('view_reports') }}">Reports</a> </li>
+                            <li><a href="{{ url_for('view_adminlog') }}">Log</a> </li>
```

Comment:
This was also suggested by staff and was added, however it wraps to a new line because of the massive search bar. Your screenshot seems to replace the About dropdown and I'd rather not have to do that. I asked for advice on how to proceed but haven't heard back yet. I am not familiar at all with css so I guess we'll have to wait until I know a fix.